### ISSUE TITLE
[L1V2] Typos

### ISSUE THREAD

**Issue by adrienrougny** [sbgn/entity-relationships#29]
• p.1
– “that can exists” → “that can exist”
• p.5
– “such as protein, a nucleic acid, metabolite” → “such as a protein, nucleic acid, metabolite”
• p.6
– “An outcome represent a particular” → “An outcome represents a particular”
• p.7
– “If A exists and B exits” → “If A exists and B exists”
– “provides three logical operators, and, or, and not” → “provides four logical operators, and,
or, not and delay”, otherwise the delay operator should be suppressed
• p.9
– “the greek letter “τ ” (“TAU”) placed” → “the label “τ ” (the greek letter, read as “TAU”)
placed” in order to be coherent with the way the other operators are described?
• p.12
– “the number of instances of an entity” → “the number of instances of an interactor”?
• p.13
– “involving only one entity, or outcomes of interactions involving one entity,” → “involving
only one interactor,”?
– “is intra-molecular . . . or complex” → is intra-molecular (i.e. between the same instance of
the interactor) or inter-molecular (i.e. between different instances of the interactor).”?
• p.15
– “the effect of an entity” → “the effect of an entity node”?
• p.16
– Figure 2.22: “Example of a stimulation a phenotype” → “Example of a stimulation tar-
geting a phenotype”?
• p.18
– “of a absolute inhibition” → “of an absolute inhibition”
• p.19
– “of a absolute stimulation” → “of an absolute stimulation”
• p.20
– “Logic arc is used to represent the fact that an interactor influences the outcome of a logic
operator.” Logic arcs can also link non-interactors (pertubing agents and logical operators)
to logical operators.
• p.22
– “a transport is therefore could be described as assignment” → “a transport is therefore
described as an assignment” or “a transport could therefore be described as an assignment”
depending on whether it is the only way to describe transport or not
• p.26
– “the semantic of the graph” → “the semantics of the graph”?
• p.27
– “In the Figure 2.41” → “In Figure 2.41”
• p.29
– “all statement on Entity Relationship map” → “all statements in/of an Entity Relationship
map”
– “is splitted to represent . . . as shown in Figure 2.45.” → “is splitted as shown in Figure
2.45”?
– “In the Figure 2.45” → “In Figure 2.45”
• p.33
– “or as a complex subunint” → “or as a complex subunit”
• p.46
– “on the on the interaction” → “on the interaction”
• p.49
– “the large number possible” → “the large number of possible”

**Comment by tczauderna** [comment]
This issue should probably be split into several issues, one for typos and one for each bug

**Comment by adrienrougny** [comment]
Indeed, and this actually only shows the typos. Changing the title to typos only.
